### PR TITLE
Expose stream executor namespace in cmake shared object.

### PR DESCRIPTION
**Pull request by gunan**

### CODE REVIEW

**Comment by gunan** [comment]
Testing here:
http://ci.tensorflow.org/view/TF%20pull%20requests/job/tf-pr-win-cmake-gpu/29/

**Comment by gunan** [comment]
Another try here
http://ci.tensorflow.org/view/TF%20pull%20requests/job/tf-pr-win-cmake-gpu/30/

**Comment by gunan** [comment]
Yet another try:
http://ci.tensorflow.org/job/tf-pr-win-cmake-gpu/31/

File: tensorflow/contrib/cmake/tools/create_def_file.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
                         r"tensorflow::|"
                         r"functor::|"
                         r"\?nsync_|"
-                        r"perftools::gputools")
+                        r"stream_executor::")
```
Comment (mrry) [inline tensorflow/contrib/cmake/tools/create_def_file.py]: This looks like the right fix, but it seems we're once again hitting the 2**16 exported symbols limit. Perhaps `stream_executor::*` contains far more symbols that `perftools::gputools` ever did?

I don't know anything about the structure of that namespace, but is it possible we could use a narrower regex to get the symbols we need here? Ideally we'd have some sort of explicit `public` or `private`/`impl` subnamespace that we could include/exclude as possible, but I'm not sure if such a thing exists...

**Comment by gunan** [comment]
I think we now got around the symbol limit issue, however now I am seeing this:
http://ci.tensorflow.org/view/TF%20pull%20requests/job/tf-pr-win-cmake-gpu/34/console
```
C:\tf_jenkins\workspace\tf-pr-win-cmake-gpu\cmake_build\Release\contrib_image_sirds_ops_gen_python.exe : fatal error LNK1169: one or more multiply defined symbols found [C:\tf_jenkins\workspace\tf-pr-win-cmake-gpu\cmake_build\contrib_image_sirds_ops_gen_python.vcxproj]
```

**Comment by gunan** [comment]
We now have a successful build!
http://ci.tensorflow.org/view/TF%20pull%20requests/job/tf-pr-win-cmake-gpu/38/

**Review by mrry — APPROVED** [review]
Hooray!

**Pull request merged**